plutus-core/plutus-core/src/PlutusCore/Parser/Type.hs

```diff
@@ -123,17 +126,21 @@ defaultUniApplication = do
 -- i.e. parse into @Tree Text@ and do the kind checking afterwards, but given that we'll still need
 -- to do the kind checking of builtins regardless (even for UPLC), we don't win much by deferring
 -- doing it.
+-- We don't support constants of type bls12_381_mlresult, so there's no case for that.
 defaultUni :: Parser (SomeTypeIn (Kinded DefaultUni))
 defaultUni = choice $ map try
     [ trailingWhitespace (inParens defaultUniApplication)
-    , someType @_ @Integer <$ symbol "integer"
```

Comment:
This definition is not about constants though, it's about types. We can't let the user spell out the type in a program?